libs/math/tests/unit/fixed_point/fixed_point.cpp

```diff
@@ -753,7 +786,7 @@ TYPED_TEST(TranscendentalTest, Exp)
   TypeParam tolerance = TypeParam::TOLERANCE;
   for (size_t i{0}; i < N; i++)
   {
-    r                = static_cast<double>(random()) / static_cast<double>(RAND_MAX);
+    r                = static_cast<double>(std::rand()) / static_cast<double>(RAND_MAX);
```

Comment:
can you just reassure me that it's fine to use std::rand() here rather than our LFG/LCG seeded by a fixed seed (i.e. isn't it possible to generate different random number here on another machine and fail the test?)

Reply:
It's possible, but these functions are well tested to be very robust and uniformly random, if our test fails then that would be an actual problem in Exp() and not the test suite.